src/mme-app/handlers/attach_authresp.c

```diff
@@ -152,6 +152,8 @@ post_to_next()
 	sec_mode_msg.enb_s1ap_ue_id = ue_entry->s1ap_enb_ue_id;
 	memcpy(&(sec_mode_msg.ue_network), &(ue_entry->ue_net_capab),
 		sizeof(struct UE_net_capab));
+	memcpy(&(sec_mode_msg.ms_net_capab), &(ue_entry->ms_net_capab),
```

Comment:
Trying to understand if ms_net_Capab always exist ? or its optional.

Reply:
It is optional. But the structure is copied from one Auth response stage to the SMC stage. There is a 'pres' flag introduced to check if any value has been received for MS network Capabilities.